bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/requirements/BaseLicensingRequirement.java

```diff
@@ -78,4 +80,50 @@ public String toString() {
 		return sb.toString();
 	}
 
+	@Override
+	public int hashCode() {
+		final int prime = 31;
```

Comment:
Please consider using `Objects.hash(featureIdentifier, featureName, ...)` instead of autogenerated method here?

Reply:
Done